packages/geoadmin-elevation-profile/src/GeoadminElevationProfilePlot.vue

```diff
@@ -0,0 +1,453 @@
+onMounted(() => {
+    window.addEventListener('beforeprint', resizeChartForPrint)
+    window.addEventListener('afterprint', resizeChart)
+
+    if (profileChartContainerRef.value) {
+        resizeObserver = new ResizeObserver(resizeChart)
+        resizeObserver.observe(profileChartContainerRef.value)
+    }
+})
+
+onUnmounted(() => {
+    window.removeEventListener('beforeprint', resizeChartForPrint)
+    window.removeEventListener('afterprint', resizeChart)
+
+    resizeObserver?.disconnect()
+})
+
+function startPositionTracking() {
+    track.value = true
+}
+function stopPositionTracking(): boolean {
+    track.value = false
+    return true
+}
+function clearHoverPosition() {
+    pointBeingHovered.value = undefined
+}
+function resetZoomToBaseValue() {
+    resetZoom(chartRef.value?.chart, 'none') // ref for chart
+}
+function resizeChartForPrint() {
+    // Here in order to have a nice PDF print of the profile we need to resize it to a fix
+    // size somehow. If we don't do this then the print is a bit deformed and pixelized.
+    // The resize to 1024x1024 is a choice that provide a nice output
+    chartRef.value?.chart.resize(1024, 1024)
+}
+function resizeChart() {
+    chartRef.value?.chart.resize()
+}
+</script>
+<template>
+    <div
+        ref="profileChartContainer"
+        class="tw:flex tw:flex-grow-1 tw:overflow-hidden tw:min-h-100px tw:p-2"
+        @mouseenter="startPositionTracking"
+        @mouseleave="stopPositionTracking"
+    >
+        <LineChart
+            v-if="chartJsOptions"
+            ref="chart"
+            :data="chartJsData"
+            :options="chartJsOptions"
+            class="tw:min-w-full"
+            data-cy="profile-graph"
+            @mouseleave="clearHoverPosition"
+            @contextmenu.prevent="resetZoomToBaseValue"
+        />
+    </div>
+    <div
+        ref="profileTooltip"
+        class="tw:fixed tw:bg-white tw:border tw:rounded tw:py-1 tw:px-2"
+        :style="tooltipStyle"
+        data-cy="profile-popup-tooltip"
+    >
+        <div
+            v-if="pointBeingHovered && pointBeingHovered.hasElevationData"
+            class="tw:p-1 tw:m-auto"
+        >
+            <div>
+                <small>
+                    <strong>{{ t('profile_x_label') }}: </strong>
+                    <span data-cy="profile-popup-tooltip-distance">
+                        {{ pointBeingHovered.dist }} {{ unitUsedOnDistanceAxis }}
+                    </span>
+                </small>
+            </div>
+            <div>
+                <small>
+                    <strong>{{ t('profile_y_label') }}: </strong>
+                    <span
+                        v-if="pointBeingHovered.elevation && pointBeingHovered.elevation > 0"
+                        data-cy="profile-popup-tooltip-elevation"
+                    >
+                        {{ pointBeingHovered.elevation }} m
+                    </span>
+                    <span v-else>{{ t('profile_not_available') }}</span>
```

Comment:
Does this take the translations from the main package? 🤔

Reply:
yes, I couldn't find a proper method to "encapsulate" the `vue-i18n` service inside a component. It latches itself to some kind of Vue global / window, and instances start mixing together. It's not yet possible to ship it as a stand-alone.

We could start looking for alternatives in the i18n space, where that would be a core feature we want (being able to be encapsulated in a component lib)